### PR TITLE
Wd 3811 dev canonical com contact us

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Rebuild Contact us as per the [design](https://warthogs.atlassian.net/browse/WD-3811)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- QA according to the new design. 

## Issue / Card

Fixes #

## Screenshots
image
image
image

### CODE REVIEW

**Comment by juanruitina** [comment]
- [ ] Please remove the breadcrumbs, as per the mockup.
- [ ] Spacing between the "Online" and "Technical assistance" rows is different than between all others.

I feel the suru doesn't work well here. Too prominent, too little content above it for it to work, a border just below. Wouldn't some space between h1 and the next row do?

**Comment by lyubomir-popov** [comment]
> I feel the suru doesn't work well here. Too prominent, too little content above it for it to work, a border just below. Wouldn't some space between h1 and the next row do?

I think that's reasonable. I'll remove it.

**Comment by lyubomir-popov** [comment]
ready for another look

**Comment by juanruitina** [comment]
Suru's still there, and there's no margin above the H1. Can you fix it?

**Review by mtruj013 — APPROVED** [review]
+1 with changes, the ones I mentioned as well as the suru and h1 margin that Juan requested

File: templates/base_index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@
     <link rel="stylesheet" type="text/css" media="screen" href="{{ versioned_static('css/styles.css') }}" />
   </head>
 
-  <body {% if is_paper %} class="is-paper" {% endif %}>
```
[the remark this answers is not in the document]
Comment (lyubomir-popov) [reply templates/base_index.html:83]: fixed

File: templates/contact-us.html [hunk 2]
```diff
@@ -7,160 +7,263 @@
 {% block meta_description %}Contact information for Canonical, the publisher of Ubuntu.{% endblock %}
 
 {% block content %}
-  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
-    <div class="p-strip--suru-image">
+
+<section class="p-section">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <div class="p-section">
```
[the remark this answers is not in the document]
Comment (lyubomir-popov) [reply templates/contact-us.html:14]: switched to a regular one

**Comment by lyubomir-popov** [comment]
the contact us page is ready for another look @juanruitina @mtruj013.

image

**Comment by juanruitina** [comment]
Demo is broken, but what I see from the screenshot looks good. So UX +1.

**Pull request merged**